src/OpenLoco/IndustryManager.cpp

```diff
@@ -190,18 +190,238 @@ namespace OpenLoco::IndustryManager
         }
     }
 
+    // 0x00459A05
+    static bool isTooCloseToNearbyIndustries(const Map::Pos2& loc)
+    {
+        for (auto& industry : industries())
+        {
+            const auto dist = Math::Vector::manhattanDistance(loc, Map::Pos2{ industry.x, industry.y });
+            if (dist < 480)
+            {
+                return true;
+            }
+        }
+        return false;
+    }
+
+    // 0x00459A50
+    static bool isOutwithCluster(const Map::Pos2& loc, const uint8_t indObjId)
+    {
+        auto numClusters = 0;
+        for (auto& industry : industries())
+        {
+            if (industry.objectId != indObjId)
+            {
+                continue;
+            }
+            numClusters++;
+            const auto dist = Math::Vector::manhattanDistance(loc, Map::Pos2{ industry.x, industry.y });
+            if (dist < 960)
+            {
+                return false;
+            }
+        }
+        if (numClusters < 3)
+        {
+            return false;
+        }
+        return true;
+    }
+
+    // 0x00469A81
+    static int16_t mountainHeight(const Map::Pos2& loc)
```

Comment:
Perhaps the tile manager would be a more appropriate location for this function?